pkg/report/utils.go

```diff
@@ -0,0 +1,112 @@
+package report
+
+import (
+	"fmt"
+	"html/template"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+)
+
+const (
+	RootReportDirectory = "reports"
+	BuildLogsDir        = "builds"
+	TestLogsDir         = "tests"
+	JunitReportDir      = "junit"
+)
+
+func InitDibReport() (*DIBReport, error) {
+	generationDate := time.Now()
+	name := generationDate.Format("20060102150405") // equivalent of `$ date +%Y%m%d%H%M%S`
+
+	// If we are in dev|test mode, generate a static report name for convenience
+	_, exist := os.LookupEnv("IS_TEST")
+	if exist {
+		name = "test_report"
+	}
```

Comment:
This is only used for local development, inside my `runConfiguration` to generate a consistent report name.

Its more convenient when i test my code and i want to see the rendered HTML in my browser (just need to reload the page, instead of looking for the new report in my filesystem)

Reply:
Ok, we might consider using env vars to configure the output path directly (something like `DIB_REPORT_DIR`), might be useful in CI as well.

Let's do that in a next PR then ;)